examples/cpp/SampleCppUWP/SampleCppUWP.vcxproj

```diff
@@ -11,6 +11,15 @@
     <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>

```

Comment:
Is this line duplicated?

Reply:
I think we can remove that.. Then the rest of the flow should be fine, i.e. the logic below should be able to auto-detect the Win 10 SDK -latest automagically. What I'm worried about is if I should test the removal of it with 2017.